<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1419</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>857</x>
      <y>0</y>
      <width>561</width>
      <height>891</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CodeEditor" name="CodeEditorUut" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="TryToCompile">
       <property name="text">
        <string>尝试编译</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ResultShower" name="CompileResult">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ResultShower" name="AI_CodeShower">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1419</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QWidget</extends>
   <header location="global">codeeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultShower</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">resultshower.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
